include/kodlab_mjbots_sdk/pid.h

```diff
@@ -0,0 +1,192 @@
+/**
+ * @file pid.h
+ * @author Chandravaran Kunjeti (kunjeti@seas.upenn.edu)
+ * @brief Class implementation of PID
+ * @date 9/30/2022
+ *
+ * @copyright 2022 The Trustees of the University of Pennsylvania. All rights reserved.
+ *
+ */
+
+#pragma once
+
+#ifndef _PID_
+#define _PID_
+
+#include "Eigen/Geometry"
+
+namespace kodlab
+{
+
+  /**
+   * @brief Pid object
+   */
+  class Pid
+  {
+
+  public:
+    /**
+     * @brief Construct a Pid object
+     * @param p_gain Proportional Gain
+     * @param d_gain Derivative Gain
+     * @param i_gain Integral Gain
+     * @param time_step Time difference between 2 updates
+     */
+    Pid(double p_gain, double d_gain, double i_gain, double time_step, double deadband, double saturation)
```

Comment:
Saturation and deadband should be optional parameters, many use-cases will not require these.

Reply:
Have made them optional variables that can be set